ApplicationProxy.cfc

```diff
@@ -0,0 +1,3 @@
+component extends="Application"{
```

Comment:
@lmajano: It's basically a convenience proxy so that I don't have to maintain a separate App.cfc for the test harness App.cfc, which duplicates everything but the paths in the core file anyway.  It also assists with path resolution for the resources directory when running tests, since we moved those sample APIs down to the module `models/resources` directory.